client/task-list/tasks/payments/stripe.js

```diff
@@ -257,6 +247,42 @@ class Stripe extends Component {
 		);
 	}
 
+	renderOauthConfig() {
+		const tosPrompt = interpolateComponents( {
+			mixedString: __(
+				'By clicking "Connect," you agree to the {{tosLink}}Terms of Service{{/tosLink}}. Or {{manualConfigLink}}manually enter your Stripe API details{{/manualConfigLink}} instead.',
+				'woocommerce-admin'
+			),
+			components: {
+				tosLink: (
+					<Link
+						href="https://wordpress.com/tos"
+						target="_blank"
+						type="external"
+					/>
+				),
+				manualConfigLink: (
+					<Link
```

Comment:
Minor, but I think semantically a `Button` with an `isLink` prop may be more appropriate here since there's no actual link.

Reply:
This makes sense. I've changed this in https://github.com/woocommerce/woocommerce-admin/pull/4933/commits/50a50b293f9a30c423c1db53ca09b391ff9f8ecb

I am seeing a change in color though: https://d.pr/i/jbpKDU

I think this is fine, but just double-checking whether there is a class we can add to get the color to match the `Link` component?

Reply:
Good eye.  The link buttons seem to be explicitly set to that color and regular links should probably be updated to match it (or vice versa), but we can tackle in a follow-up.